docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,180 @@
+
+## Alerting
+
+_All Alerting improvements are generally available in all editions of Grafana._
+
+### Search for alert rules from multiple data sources
+
+Search for and display alert rules for multiple data sources at the same time.
+
+### Fuzzy search on the Alert rule list view
+
+Search for namespaces or folders, evaluation groups, and alert rule names on the Alert rules list view with immediate results, and regardless of typos.
+
+### Access an alert rule from a dashboard or a panel
+
+Navigate to an alert rule directly from a dashboard or a panel to easily access the alert rule details.
+
+{{< figure src="/media/docs/alerting/alert-rule-dashboard.png" max-width="750px" caption="Access an alert rule from a dashboard or a panel" >}}
+
+### Access a dashboard or panel from an alert rule
+
+Navigate from an alert rule straight to a dashboard or a panel associated with the alert rule to visualize your alerting data.
+
+{{< figure src="/media/docs/alerting/dashboard-alert-rule.png" max-width="750px" caption="Access a dashboard or panel from an alert rule" >}}
+
+### Preview queries for recording rules
+
+Visualize queries when creating or editing recording rules, so you can see the results of your query before saving your recording rule.
+
+{{< figure src="/media/docs/alerting/preview-queries-recording-rule.png" max-width="750px" caption="Preview queries for recording rules" >}}
+
+### Wait before firing alerts when no data is returned
+
+Alert rules that are configured to fire when an evaluation returns no data now only fire when the entire duration of the evaluation period has finished. This means that rather than immediately firing when the alert rule condition is breached, the alert rule waits until the time set in the **For** field has finished and then fires, reducing alert noise and allowing for temporary data availability issues.
+
+### Improved Notification policies view
+
+Updates to the Notification policies view make it easier to use and manage in the following ways:
+
+- View default policy and nested policies at a glance
+- New tab for mute timings
+- View alert instances for each policy
+- View contact points and which integrations are configured for each policy
+- View inherited properties on nested policies
+- Search for label matchers and for contact points to see which notifications are going where
+
+{{< figure src="/media/docs/alerting/notification-policies-view.png" max-width="750px" caption="Improved Notification policies view" >}}
+
+### Guidance for configuring your Alertmanager
+
+Get additional help while configuring your Alertmanager. If you enter an invalid Alertmanager configuration, an error message displays, and you can choose from a previous working configuration to restart it.
+
+## InfluxDB plugin database field deprecation
+
+The `database` field in the provisioning file has been deprecated.
+This information will be stored in `jsonData` with `dbName` field.
```

Comment:
@itsmylife - Can we make this edit?

```suggestion
This information will be stored in the `jsonData` file using the `dbName` field.
```

Reply:
I will make that change like the following: 
```
This information will be stored in the `jsonData` field using the `dbName` property.
```

Reply:
Fixed here https://github.com/grafana/grafana/pull/64168/commits/9c6a4df961be5a6cac84009baf836e13a1b65b95